src/Raven.Studio/typescript/viewmodels/manage/certificates.ts

```diff
@@ -27,6 +27,9 @@ interface unifiedCertificateDefinitionWithCache extends unifiedCertificateDefini
     expirationClass: string;
     expirationText: string;
     expirationIcon: string;
+    validFromClass: string;
+    validFromText: string;
+    validFromIcon: string;
```

Comment:
put not before, not after here as number (moment.utc(date).getDate().getTime()) and use for compare function

Reply:
done